Update Mail public folders are in Preview

dakoer opened the pull request:
Starting in AAD Connect v1.1.524.0, syncing Mail-Enabled Public Folders is now in preview.

https://docs.microsoft.com/en-us/azure/active-directory/connect/active-directory-aadconnect-version-history#115240

azurecla commented:
Hi __@JasonMAnderson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (janders). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dakoer committed:
Update Mail public folders are in Preview

Starting in AAD Connect v1.1.524.0, syncing Mail-Enabled Public Folders is now in preview.

https://docs.microsoft.com/en-us/azure/active-directory/connect/active-directory-aadconnect-version-history#115240
